crates/sparrow-session/src/tests.rs

```diff
@@ -0,0 +1,2 @@
+//! Partitioned execution tests
+mod partitioned;
```

Comment:
Probably unnecessary levels of nesting -- I'd just put the tests in `tests.rs` for now -- we're testing "tat sessions work" or "`tests/execution.rs` -- we're testing that "sessions can execute expressions".

Reply:
Alternatively, move this to `src/partitioned/tests.rs` rather than `src/tests.rs`.